### PR TITLE
Bump Arm build to ARMv8

### PR DESCRIPTION
**Pull request by haslinghuis**
Fixes: https://github.com/betaflight/betaflight-configurator/issues/2882

https://news.ycombinator.com/item?id=27277351

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by McGiverGim — COMMENTED** [review]
I'm not too sure if we need to remove armv7 support or support both. The only difference I suppose is the downloadUrl?

**Comment by haslinghuis** [comment]
Raspberry Pi 3 can run on ARMv8 kernel not sure about other platforms.

**Comment by McGiverGim** [comment]
The original PR is here: https://github.com/betaflight/betaflight-configurator/pull/1024

@nmaggioni please, can you help with this?

**Comment by nmaggioni** [comment]
The latest NW.JS community builds should be compatible with both ARMv7 and v8, but [as far as I understood](https://community.arm.com/support-forums/f/architectures-and-processors-forum/4798/armv8-backwards-compatibility-with-armv7/11879#11879) (and already linked to HN by @haslinghuis) it will be down to the specific hardware implementations to properly support them or not.
I'd say go for it and drop the v7-only build, it's not like it's a huge changeset to monkeypatch back in if needed.

File: gulpfile.js
Comment (nmaggioni) [inline gulpfile.js:515]: Hardcoding SDK URLs deep down in the build code is almost never a good idea, especially if prospecting to have multiple builds branch out of this one. I'd keep the modular URL approach even if needs to be simplified to a single global var.

**Review by McGiverGim — APPROVED** [review]

**Review by limonspb — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Pull request merged**
